templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li> User: <a href="{% url "taxi:driver-detail" pk=user.id %}"> {{ user.username }}</a></li>
```

Comment:
The link to the user's detail page uses the `user.id` as the primary key. Ensure that the `Driver` model uses `id` as the primary key, or adjust the URL to use the correct field if a different primary key is used.

Reply:
Everything works for me. According to the assignment.